src/vm/clrconfiguration.cpp

```diff
@@ -0,0 +1,21 @@
+// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
Nit: It may be nice to call it clrconfignative.* - the "native" suffix is used to differentiate the file with FCalls/QCalls from the regular impl file.